backend/src/main/java/org/hackbrooklyn/plaza/security/Roles.java

```diff
@@ -5,34 +5,39 @@
 import org.springframework.stereotype.Component;
 
 import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 @Component
 public class Roles {
     public static final String ADMIN = "ROLE_ADMIN";
     public static final String VOLUNTEER = "ROLE_VOLUNTEER";
     public static final String PARTICIPANT = "ROLE_PARTICIPANT";
+    public static final String APPLICANT = "ROLE_APPLICANT";
+    public static final String NONE = "ROLE_NONE";
 
     public static final Set<GrantedAuthority> ADMIN_GRANTED_AUTHORITIES;
     public static final Set<GrantedAuthority> VOLUNTEER_GRANTED_AUTHORITIES;
     public static final Set<GrantedAuthority> PARTICIPANT_GRANTED_AUTHORITIES;
+    public static final Set<GrantedAuthority> APPLICANT_GRANTED_AUTHORITIES;
+    public static final Set<GrantedAuthority> NONE_GRANTED_AUTHORITIES;
 
+    // Each role inherits the role directly under it, with the exception of Admin
+    // Admin has its own unique set of roles that supersedes all of the other roles
+    // and effectively serves as a listing of all available roles in each category
     static {
         // Admins
-        String[] adminAuthorities = {
-                // Applications
-                Authorities.APPLICATIONS_READ,
-                Authorities.APPLICATIONS_UPDATE_DECISION,
-                Authorities.APPLICATIONS_DELETE,
-
+        List<String> adminAuthorities = Arrays.asList(
                 // Announcements
                 Authorities.ANNOUNCEMENTS_CREATE,
-                Authorities.ANNOUNCEMENTS_READ,
+                Authorities.ANNOUNCEMENTS_READ_PUBLIC,
```

Comment:
Shouldn't Admin also be given the ANNOUNCEMENTS_READ_PARTICIPANTS_ONLY authority

Reply:
Yep! Latest push should fix this issue.